### PR TITLE
Added D&M team members to about page

### LINKED ISSUE #390: Add new D&M team members to about page
- photos: https://drive.google.com/drive/u/2/folders/1lZV6wTIOqNPP6CpUmpY-wJOiFUQtcmP4
- roster: https://docs.google.com/spreadsheets/d/1p8LcLM_FijHSsT5unc6pWjH8aHHbVn2meK65hfmRIKo/edit

### PR DESCRIPTION
**Pull request by andrewmniu**
## Status
- Done

## GitHub Issues addressed (`#IssueNumber`)
- This PR closes #390 

## What I did
- I added the new members from D&M to the about page

## Screenshots
- Before
image
- After
image

## Tests
- n/a

## Questions/Discussions/Notes
- Anne Yong was listed on the 2021 team roster, but I could not find her image in the image folder.
- I'm not sure why, but git diff shows that I deleted the entire json file and then re-added everything

## Merging the PR
- Who should merge this PR?
  - [ ] I will merge it myself
  - [x] The last reviewer to approve can merge it
- Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
  - [ ] preserve the history
  - [x] squash-merge

### CODE REVIEW

**Review by jackliu612 — COMMENTED** [review]
Awesome job, thanks for taking care of this for us. We can see what to do about Anne's missing pfp. It is weird how the git diff worked out but the changes look good to me as far as I can tell.

File: tcf_website/views/team_info.json [hunk 1]
```diff
@@ -148,13 +148,6 @@
       "class": "2021",
       "img_filename": "ENG_James_Lim.jpg",
       "github": "yl2vv"
-    },
-    {
-      "name": "Kartik Chugh",
-      "role": "Developer",
-      "class": "2023",
-      "img_filename": "ENG_Kartik_Chugh.jpg",
-      "github": "KartikChugh"
```
Comment (j-alicia-long) [inline tcf_website/views/team_info.json:157]: I think you accidentally deleted Kartik lol might wanna fix that

**Review by j-alicia-long — CHANGES_REQUESTED** [review]
Looks good! Just a note about Kartik getting deleted lol

Also I think Anne finally uploaded her pic (you might have to crop it to square tho). https://drive.google.com/drive/u/0/folders/1lZV6wTIOqNPP6CpUmpY-wJOiFUQtcmP4

I'll approve this after you make those changes!

File: tcf_website/views/team_info.json (hunk 1 above)
Comment (andrewmniu) [reply tcf_website/views/team_info.json:157]: oh yeah, I'll definitely fixed that "accident"

**Review by j-alicia-long — APPROVED** [review]

**Pull request merged**